tdrs-backend/tdpservice/search_indexes/admin/filters.py

```diff
@@ -94,26 +42,26 @@ def queryset(self, request, queryset):
         return queryset
 
 
-class STTFilter(MultipleChoiceListFilter):
-    """Simple filter class to show records based on stt."""
+class STTFilter(MultiSelectDropdownFilter):
+    """Multi-select dropdown filter class to filter records based on stt."""
 
-    title = _('STT Code')
+    def __init__(self, field, request, params, model, model_admin, field_path):
+        super(MultiSelectDropdownFilter, self).__init__(field, request, params, model, model_admin, field_path)
+        self.lookup_choices = self._get_lookup_choices(request)
 
-    parameter_name = 'stt_code'
+    def _get_lookup_choices(self, request):
+        """Filter queryset to guarentee lookup_choices only has STTs associated with the record type."""
+        record_type = str(request.path).split('/')[-2]
+        queryset = STT.objects.all()
+        if 'tribal' in record_type:
+            queryset = queryset.filter(type=STT.EntityType.TRIBE)
+        elif 'ssp' in record_type:
+            queryset = queryset.filter(ssp=True)
+        else:
```

Comment:
@elipe17 is this the line of code that populates the STT list for TANF? Asking because we noticed that the territories are missing from the TANF dropdown. can we get those added?